### PR TITLE
fix(tools): windows compatibility

### PR DESCRIPTION
**Pull request by Arathy-s**
## What I did

1. Added fixes to resolve Windows OS compatibility issues.

## Testing Instructions

1. The code should run in all three OS( Windows, Linux, Mac).

## Notes to Reviewers

1. Please run this code on local and check if its working.

### CODE REVIEW

File: tools/pfe-tools/dev-server/plugins/import-map-generator.ts [hunk 1]
```diff
@@ -29,15 +29,16 @@ async function resolveMonorepoPackages() {
...
 
   const packages = new Map();
 
-  for (const dir of [cwd, ...potentialPackageDirs]) {
-    const pkgPath = join(dir, 'package.json');
-    if (await exists(pkgPath)) {
-      const { name } = JSON.parse(await readFile(pkgPath, 'utf-8'));
-      packages.set(name, dir);
+  for (const dir of ['.', ...potentialPackageDirs]) {
+    const pkgDir = join(cwd, dir)
```
Comment (bennypowers) [inline tools/pfe-tools/dev-server/plugins/import-map-generator.ts:37]:
```suggestion
    const pkgDir = join(cwd, dir);
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -205,5 +205,26 @@ jobs:
           close-previous: true
           title: "🧪 Tests are failing on main"
           body: "It looks like the build is currently failing on the main branch. See failed [action results](https://github.com/patternfly/patternfly-elements/actions/runs/${{ github.run_id }}) for more details."
+  
+  build-windows:
```
Comment (bennypowers) [inline .github/workflows/tests.yml:209]:
```suggestion
  build-windows:
    name: Verify that build runs on Windows
```

**Review by bennypowers — APPROVED** [review]
Legacy Thinking Made Great!

Well done, @Arathy-s

**Comment by Arathy-s** [comment]
> Legacy Thinking Made Great!
> 
> Well done, @Arathy-s

Thank you so much for the support @bennypowers!!!

**Pull request merged**